SECURITY.md

```diff
@@ -0,0 +1,9 @@
+## Report a security issue
+
+The faslty-template-rust-default project team welcomes security reports and is committed to providing prompt attention to security issues. Security issues should be reported privately via [Fastly’s security issue reporting process](https://www.fastly.com/security/report-security-issue).
```

Comment:
Don't mention repo specific values and then you won't have copy paste errors :-)

```suggestion

The project team welcomes security reports and is committed to giving prompt attention to security issues. Security issues should be reported privately via [Fastly’s security issue reporting process](https://www.fastly.com/security/report-security-issue).
```